InternxtDesktop/AppDelegate.swift

```diff
@@ -501,4 +506,8 @@ class AppDelegate: NSObject, NSApplicationDelegate , PKPushRegistryDelegate {
         }
 
     }
+    
+    private func updateStorage() {
```

Comment:
`updateStorage` is not correct, we are updating the user usage, so `updateUsage` is the correct term here.